libs/SalesforceSDKCore/SalesforceSDKCoreTests/PushNotificationManagerTests.swift

```diff
@@ -645,6 +645,90 @@ class PushNotificationManagerTests: XCTestCase {
         XCTAssertEqual(mockUserAccount.notificationTypes?.count, 1)
         XCTAssertEqual(mockUserAccount.notificationTypes?.first?.apiName, "cached_type")
     }
+
```

Comment:
Could we have a test / example code showing a specific action being filtered out?

Reply:
Maybe a helper method to create a NotificationType from an existing NotificationType but keeping only specific actions?